clients/js/test/setAndVerifyCollection.test.ts

```diff
@@ -0,0 +1,66 @@
+import { createNft } from '@metaplex-foundation/mpl-token-metadata';
+import {
+  generateSigner,
+  percentAmount,
+  publicKey,
+} from '@metaplex-foundation/umi';
+import test from 'ava';
+import {
+  fetchMerkleTree,
+  getCurrentRoot,
+  hashLeaf,
+  hashMetadataCreators,
+  hashMetadataData,
+  setAndVerifyCollection,
+} from '../src';
+import { createTree, createUmi, mint } from './_setup';
+
+test('it can set and verify the collection of a minted compressed NFT', async (t) => {
+  // Given a Collection NFT.
+  const umi = await createUmi();
+  const collectionMint = generateSigner(umi);
+  await createNft(umi, {
+    mint: collectionMint,
+    name: 'My Collection',
+    uri: 'https://example.com/my-collection.json',
+    sellerFeeBasisPoints: percentAmount(5.5), // 5.5%
+    isCollection: true,
+  }).sendAndConfirm(umi);
+
+  // And a tree with a minted NFT that has no collection.
+  const merkleTree = await createTree(umi);
+  let merkleTreeAccount = await fetchMerkleTree(umi, merkleTree);
+  const leafOwner = generateSigner(umi).publicKey;
+  const { metadata, leafIndex } = await mint(umi, { merkleTree, leafOwner });
+
+  // When the tree authority verifies the collection.
```

Comment:
It might be good to point out in a comment that the collection and the tree authority are both using the default Umi authority, since otherwise a tree authority would not be the collection verifier.  Not required though, just extra info I had to ask about as i ramp up on Umi.  Maybe its too fundamental and you just need to know that to do stuff with Umi since it often defaults to that authority.

Reply:
Great point, I've updated the PR. 🙏